js/plugins-menu.js

```diff
@@ -0,0 +1,4 @@
+//Opens Upgrade link in plugins menu as new tab.
```

Comment:
This is missing Documentation for the File Header.
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#file-headers

Reply:
Done.